resolver/resolver.cc

```diff
@@ -363,14 +363,22 @@ class ResolveConstantsWalk {
         }
 
         core::SymbolRef resolved;
+        const auto stubResolved = [&resolved, &job]() {
```

Comment:
Could this be a proper method that takes 2 arguments rather than a lambda?

Reply:
How do you like af8ec812e71311f5379b8f26d09b23595eea6309?